src/metrics/metric_types.go

```diff
@@ -20,3 +24,61 @@ func (qd QueryDefinition) GetDataModels() interface{} {
 	ptr := reflect.New(reflect.ValueOf(qd.dataModels).Type())
 	return ptr.Interface()
 }
+
+func (qd *QueryDefinition) insertDatabaseNames(databases collection.DatabaseList) *QueryDefinition {
+	databaseList := ""
+	for database := range databases {
+		databaseList += `'` + database + `',`
```

Comment:
maybe use sprintf just to be consistent with the other methods?

Reply:
Done 👍